app/index.jsx

```diff
@@ -17,10 +32,11 @@ let App = React.createClass({
     this.setState({ selectedElementIndex: clickedLiIndex })
   },
   render: function () {
-    let featureTitle = constants.elements[this.state.selectedElementIndex]
+    const featureTitle = constants.elements[this.state.selectedElementIndex]
+    const Component = componentCode[this.state.selectedElementIndex]
```

Comment:
That will be holding any one of the component code at a time. So I went with generic name. Should I change it?